tutorials/build_systems/easybuild/eb_test.py

```diff
@@ -0,0 +1,25 @@
+import reframe as rfm
+import reframe.utility.sanity as sn
+
+
+class EasybuildMixin(rfm.RegressionTest):
+    @rfm.run_before('run')
+    def prepare_run(self):
+        self.modules = self.build_system.generated_modules
+
+
+@rfm.simple_test
+class BZip2Check(EasybuildMixin):
+    def __init__(self):
+        self.descr = 'This demonstrates the EasyBuild build system.'
+        self.valid_systems = ['daint:gpu']
+        self.valid_prog_environs = ['gnu']
+        self.modules = ['daint-gpu',
+                        'EasyBuild-custom']
+        self.build_system = 'EasyBuild'
+        self.build_system.easyconfigs = ['bzip2-1.0.6.eb']
+        self.build_system.options = ['-f']
+        self.sanity_patterns = sn.assert_found(r'Version 1.0.6',
+                                               self.stderr)
+        self.executable = 'bzip2'
+        self.executable_opts = ['--help']
```

Comment:
Use the new syntax without the `__init__` method here.

Reply:
Also I'd use `--version` instead of `--help`.

Reply:
Here I put `--help` because `bzip2 --version` crashes for bzip2-1.0.6. With bzip2-1.0.8 it doesn't crash, but that version doesn't install with easybuild because of a problem fetching the sources.
```
bzip2: I won't write compressed data to a terminal.
bzip2: For help, type: `bzip2 --help'.
```

Reply:
Ok, we can update the tutorial later on.